core/src/main/java/bisq/core/api/CoreDisputeAgentsService.java

```diff
@@ -99,6 +99,7 @@ void registerDisputeAgent(String disputeAgentTypeString, String registrationKey)
                     ecKey = refundAgentManager.getRegistrationKey(registrationKey);
                     signature = refundAgentManager.signStorageSignaturePubKey(Objects.requireNonNull(ecKey));
                     registerRefundAgent(nodeAddress, languageCodes, ecKey, signature);
+                    break;
```

Comment:
For symmetry I would probably have a return here, or break in the MEDIATOR case. Not serious, but it makes more sense when similar cases are handled the same way. In this case one has to wonder why there is a return in one case and break in the next.

Reply:
I did not make sense to me, but codacy wanted a break (statement).   I'll revert it.

Reply:
My two ducat's worth... I think Intellij's code inspector is more intelligent than codacy's.